norm/arrayNormalization.pl

```diff
@@ -249,9 +249,9 @@ sub makeArraysToAssaysToFiles {
 			next;
 		}
 
-		# Get technology from Array design file using peach API. 
-		my $adfInfoUrl = $atlasSiteConfig->get_arrayexpress_adf_info_url;
-		my $arrayDataTech=`curl -s $adfInfoUrl$arrayDesign`;
+		# Get technology from Array design file using BioStudies API. 
+		# NB: This is a bit of a hack, but it's the only way to get the technology
+		my $arrayDataTech=`curl -s https://www.ebi.ac.uk/biostudies/api/v1/studies/$exptAccession`;
```

Comment:
You are right @anilthanki 

I tried something like 
```curl https://www.ebi.ac.uk/biostudies/api/v1/arrays/`curl -s https://www.ebi.ac.uk/biostudies/api/v1/studies/E-MTAB-8600 | jq -r  -c '.section?.subsections[] | objects | select (.type=="Assays and Data") | .subsections[] | select (.type=="Array Designs") | .links[][].url '`  -s  | jq -r '.section.attributes[] | select(.name | contains("Title")).value'```

and it worked, but I was trying to avoid to add the `jq` dependency and the complex command above, but not sure how to do that at the moment

Reply:
As far as I can remember Perl do have JSON parser.. let me find

Reply:
This might be useful, https://www.educba.com/perl-json/

Reply:
I may be able to help, if you let me know which fields we need to fetch